specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -6359,6 +6398,11 @@
           "description": "The ID of the subscription for which the invoice is generated.",
           "type": "string",
           "readOnly": true
+        },
+        "rebillDetails": {
+          "description": "Details of an Enterprise Agreement invoice's rebill relationship to other invoices.",
```

Comment:
I guess we can simplify it - Rebill details of an Enterprise Agreement invoice.

Reply:
I expect that this should work for modern as well